roles/openshift_node/tasks/upgrade/containerized_upgrade_pull.yml

```diff
@@ -10,6 +10,6 @@
     docker pull {{ osn_ovs_image }}:{{ openshift_image_tag }}
   register: pull_result
   changed_when: "'Downloaded newer image' in pull_result.stdout"
-  when: openshift_use_openshift_sdn | bool
+  when: openshift_use_openshift_sdn | default(True) | bool
```

Comment:
This looks like it should be 'openshift_node_use_openshift_sdn' and was missed in a refactor.

Reply:
Right, its a much cleaner solution. Updated the PR